config/initializers/sentry.rb

```diff
@@ -8,6 +8,7 @@
   config.traces_sample_rate = 0.05
   # crash-free sessions tracking
   config.auto_session_tracking = true
+  config.release = ENV["SOURCE_VERSION"] || "development"
```

Comment:
nitpick: pour + de clarté je proposerais de faire:
```suggestion
  config.auto_session_tracking = true
  # "SOURCE_VERSION" env variable is set automatically by Scalingo as the SHA of the commit
  config.release = ENV["SOURCE_VERSION"] unless Rails.env.local?
```